source/Engine.cpp

```diff
@@ -1387,8 +1387,6 @@ void Engine::CalculateStep()
 		return;
 
 	// Handle the mouse input of the mouse navigation
-	if(Preferences::Has("alt-mouse turning") && !isMouseTurningEnabled)
-		activeCommands.Set(Command::MOUSE_TURNING_TOGGLE);
```

Comment:
This may need a better way to load on game start.  What do you recommend?  Feels weird to try to restore preference state every frame.

Maybe it should set the value to the Has check every time.

Reply:
```
isMouseTurningEnabled = Preferences::Has("alt-mouse turning");
```

Reply:
Maybe initialize `isMouseToggleEnabled` with the preference in the constructor? I think

Reply:
Fix pushed